### PR TITLE
issue fixed

### PR DESCRIPTION
**Pull request by ankushtripathii**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1723"></a>

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

File: public/data/ankushtripathii.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "url": "https://github.com/ankushtripathii",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/imalphaorionis",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/ankushtripathii.json:18]: I recommend capitalizing the `i` here. What do you think? :)
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Comment (ankushtripathii) [reply public/data/ankushtripathii.json:18]: > 

yup right thanks

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/1723/conflicts).
Let us know if you need any help with that! 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**